apps/ledger-live-mobile/src/screens/Swap/Form/index.tsx

```diff
@@ -428,10 +386,5 @@ export function SwapForm({
     );
   }
 
-  // Sentry?
```

Comment:
Idem here

Reply:
Since we are no longer using the providers endpoint to get the providers this bit of code seemed redundant.

Reply:
Ok, I think then you need to change the logic in the top:

```
if (storedProviders?.length) {....} <Loading>
```